inspirehep/modules/records/receivers.py

```diff
@@ -106,3 +114,24 @@ def match_valid_experiments(recid, json, *args, **kwargs):
                     # If the experiment cannot be matched it is considered valid
                     facet_experiment = exp.get("experiment")
                 exp.update({"facet_experiment": facet_experiment})
+
+
+@before_record_insert.connect
+@before_record_update.connect
+def dois_clean(sender, *args, **kwargs):
+    dois = []
+    new_l = []
+    if 'dois' in sender:
+        for element in sender['dois']:
```

Comment:
And `element['value']` will always be a string with the doi? Or we will get a suprise and get a list or something ?